examples/java/unit-test/rest/TwilioRestTest.java

```diff
@@ -264,6 +264,30 @@ public void testObjectArrayTypeParam() {
         assertNotNull(awsCreator);
     }
 
+    @Test
+    public void testAnyTypeParam() {
+        Request mockRequest = new Request(
+                HttpMethod.POST,
+                "api",
+                "/v1/Credentials/AWS"
+        );
+
+        Map<String, Object> anyMap = new HashMap<>();
+        anyMap.put("TestInteger", 1);
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+        mockRequest.addPostParam("TestAnyType", "{TestInteger=1}");
+        mockRequest.addPostParam("TestString", "AC222222222222222222222222222222");
+        when(twilioRestClient.request(mockRequest)).thenReturn(new Response("{\"account_sid\":\"AC222222222222222222222222222222\", \"sid\":\"PNXXXXY\"}", 200));
+        when(twilioRestClient.getObjectMapper()).thenReturn(objectMapper);
+        AwsCreator awsCreator = new AwsCreator("AC222222222222222222222222222222");
+        awsCreator.setTestAnyType(anyMap);
+        awsCreator.create(twilioRestClient);
```

Comment:
fetch the response  for ` awsCreator.create(twilioRestClient)` (Account object) and assert that for not null .

Reply:
Thanks for pointing! 
Updated!